lib/matplotlib/collections.pyi

```diff
@@ -84,8 +84,8 @@ class PathCollection(_CollectionWithSizes):
         num: int | Literal["auto"] | ArrayLike | Locator = ...,
         fmt: str | Formatter | None = ...,
         func: Callable[[ArrayLike], ArrayLike] = ...,
-        **kwargs
-    ): ...
+        **kwargs,
+    ) -> tuple[list[Artist], list[str]]: ...
```

Comment:
```suggestion
    ) -> tuple[list[Line2D], list[str]]: ...
```

According to the docstring this _can_ be more specific.

Would require an import as well.